src/action/wallet.js

```diff
@@ -449,6 +453,20 @@ class WalletAction {
     }
   }
 
+  /**
+   * Fetch the limbo balance using the lnd grpc api and set the
+   * corresponding value on the global store.
+   * @return {Promise<undefined>}
+   */
+  async getLimboBalance() {
+    try {
+      const r = await this._grpc.sendCommand('PendingChannels');
+      this._store.limboBalanceSatoshis = r.totalLimboBalance;
+    } catch (err) {
+      log.error('Getting limbo balance failed', err);
+    }
+  }
+
```

Comment:
`store.limboBalanceSatoshis` should be done in `getPendingChannels` of channel actions as that grpc api is already being polled. Doing it here would result in the api being polled twice.

Reply:
fixed